src/lib/model/Hub.ts

```diff
@@ -216,6 +238,37 @@ export class Hub extends HalResource {
       list: (options?: Pageable & Sortable): Promise<Page<Webhook>> =>
         this.fetchLinkedResource('webhooks', options, WebhooksPage),
     },
+
+    workflowStates: {
+      /**
+       * Creates a Workflow State inside this Hub
+       * @param resource
+       */
+      create: (resource: WorkflowState): Promise<WorkflowState> =>
+        this.createLinkedResource(
+          'create-workflow-state',
+          {},
+          resource,
+          WorkflowState
+        ),
+
+      /*/!**
+       * Get a Workflow State inside this hub by its id
+       *!/
+      get: (id: string): Promise<WorkflowState> =>
+        this.client.fetchResource(`workflow-states/${id}`, WorkflowState),
+*/
```

Comment:
If this is not required, please remove it

Reply:
Removed